### DISCUSSION TITLE
Support for strip_prefix in new_local_repository

### DISCUSSION THREAD

**Discussion by fquiers** [bazelbuild/bazel#18258] [Ideas]
Could `new_local_repository` support `strip_prefix`, like `new_git_repository` does, please?

**Comment by Wyverald** [comment]
I don't think this makes sense? If you want `new_local_repository(path="foo/bar", strip_prefix="abc/def")`, why not just write `new_local_repository(path="foo/bar/abc/def")`?

**Comment by lberki** [comment]
Given @Wyverald 's suggestion that seems to be a principled and nice alternative to what you recommend, I'll close this thread. Please re-open if you think it's somehow unsatisfactory.
